### PR TITLE
GIX-1115: Stake SNS neuron functionality

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

User is able to stake SNS neurons.

# Changes

* New sns api function: stakeNeuron.
* Implement sns neurons service: stakeNeuron.
* Add governanceCanisterId to SnsSummary type.
* Change how slot "destination-info" is rendered inside TransactionReview.

# Tests

* Test for new sns api function.
* Test for new sns neurons service.
* Fix broken tests.

### CODE REVIEW

**Comment by lmuntaner** [comment]
New UX:

https://user-images.githubusercontent.com/4550653/201167307-cd25c85f-571c-4a1d-922d-ea631416af5c.mov

Changes in the Review screen. We use the SNS Governance canister plus "Subaccount of..."
Screenshot 2022-11-10 at 18 39 04

**Comment by lmuntaner** [comment]
@peterpeterparker @mstrasinskis please review

File: frontend/src/lib/modals/sns/StakeSnsNeuronModal.svelte [hunk 1]
```diff
@@ -49,7 +49,7 @@
       toastsSuccess({
         labelKey: "sns_neurons.stake_sns_neuron_success",
         substitutions: {
-          $$tokenSymbol: token.symbol,
```
Comment (mstrasinskis) [inline frontend/src/lib/modals/sns/StakeSnsNeuronModal.svelte:52]: was it a typo?

**Review by mstrasinskis — APPROVED** [review]
LGTM, thank you!

File: frontend/src/lib/modals/sns/StakeSnsNeuronModal.svelte (hunk 1 above)
Comment (lmuntaner) [reply frontend/src/lib/modals/sns/StakeSnsNeuronModal.svelte:52]: yes

**Pull request merged**